src/tests/ftest/util/dmg_utils_params.py

```diff
@@ -25,17 +26,36 @@ def _get_new(self):
         return DmgTransportCredentials(self._log_dir)
 
 
+class DmgTelemetryCredentials(TelemetryCredentials):
+    """Telemetry credentials listing certificates for secure communication."""
+
+    def __init__(self, log_dir="/tmp"):
+        """Initialize a TelemetryCredentials object."""
+        super().__init__("/run/dmg/telemetry_config/*", None, log_dir)
+        self.ca_cert = LogParameter(self._log_dir, None, "daosTelemetryCA.crt")
+
+    def _get_new(self):
+        """Get a new object based upon this one.
+
+        Returns:
+            DmgTelemetryCredentials: a new DmgTelemetryCredentials object
+        """
+        return DmgTelemetryCredentials(self._log_dir)
```

Comment:
I don't think this root cert is something we should be passing in to dmg. We need to think about these certs a bit differently than the DAOS component certs, which are used for authentication as well as transport security. For our HTTPS telemetry endpoint, we are only using it for transport security, i.e. to prevent an attacker from injecting something between the telemetry endpoint and whatever tool is being used to read the telemetry (like dmg or prometheus, for example).

Here is [an explainer for how self-signed certs work for HTTPS](https://stackoverflow.com/a/65432506). The admin will need to mark the root certificate as trusted in each client node's OS. If they do this, I would expect the HTTPS endpoint to work without further configuration on their part. The admin will be controlling these certificates, so will probably be already doing this.

Reply:
I will have to check the tool and change the code.